website/pages/index.tsx

```diff
@@ -1,5 +1,327 @@
 import React from 'react';
+import { useRouter } from 'next/router';
+import { css, cx } from 'emotion';
+import facepaint from 'facepaint';
+import { Overline } from '@leafygreen-ui/typography';
+import { uiColors } from '@leafygreen-ui/palette';
+import { useViewportSize } from '@leafygreen-ui/hooks';
+import { spacing, breakpoints } from '@leafygreen-ui/tokens';
+import { GridContainer, GridItem } from 'components/Grid';
+import News from 'components/News';
+
+const mq = facepaint(
+  Object.values(breakpoints).map(bp => `@media (min-width: ${bp}px)`),
+  { literal: true },
+);
+
+const backdrop = css`
+  background-color: ${uiColors.gray.light3};
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: -1;
+`;
+
+const layoutProperties = css`
+  ${mq({
+    width: ['calc(100% + 48px)', '100%', '100%', '1077px'],
+    paddingRight: [0, `${spacing[4]}px`, `${spacing[4]}px`, `${spacing[4]}px`],
+    marginLeft: ['-24px', 'unset', 'unset', 'unset'],
+  })}
+`;
+
+const boxShadow = css`
+  box-shadow: 0 0 0 1px ${uiColors.gray.light1};
+`;
+
+const container = css`
+  ${boxShadow}
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  border: unset;
+  padding: unset;
+  cursor: pointer;
+  position: relative;
+  color: ${uiColors.gray.dark3};
+`;
+
+const sharedHoverInteraction = css`
+  &:hover {
+    background-color: ${uiColors.gray.light3};
+    border: 1px solid ${uiColors.gray.light1};
+    box-shadow: 0 0 0 0px ${uiColors.gray.light1},
+      2px 16px 20px -10px rgba(0, 0, 0, 0.2);
+    transform: scale(1.05);
+    z-index: 99999;
+  }
+`;
+
+const previewWrapper = css`
+  ${container}
+  overflow: hidden;
+  transition: transform 300ms ease-in-out;
+  ${sharedHoverInteraction}
+
+  &:hover {
+    & > div {
+      opacity: 1;
+    }
+  }
+`;
+
+const overlineContainer = css`
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  padding-left: ${spacing[3]}px;
+  padding-bottom: ${spacing[3]}px;
+  transition: opacity 300ms ease-in-out;
+
+  ${mq({
+    opacity: [1, 1, 0, 0],
+  })}
+`;
+
+const overlineColor = css`
+  color: ${uiColors.gray.dark1};
+`;
+
+const marketingWrapper = css`
+  ${boxShadow}
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  position: relative;
+  transition: transform 300ms ease-in-out;
+  ${sharedHoverInteraction}
+`;
+
+const textWrapper = css`
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  font-weight: medium;
+  padding-top: ${spacing[4]}px;
+  padding-left: ${spacing[4]}px;
+  text-align: left;
+  overflow: hidden;
+
+  ${mq({
+    fontSize: ['24px', '60px', '60px', '60px'],
+  })}
+`;
+
+const lgHeight = css`
+  ${mq({
+    height: ['calc(100vw / 2)', '350px', '350px', '350px'],
```

Comment:
is this not the same as
```suggestion
    height: ['calc(50vw )', '350px', '350px', '350px'],
```

Reply:
No it is, sorry.. there was a fancier calc at one point but..

Reply:
👍